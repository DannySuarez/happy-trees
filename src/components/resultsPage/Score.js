import React from 'react';
import PropTypes from 'prop-types';

const Score = ({ nickName, score }) => {

  return (
    <div>
      <h3>{nickName}</h3>
      <h3>{score}</h3>
    </div>
  );
};

Score.propTypes = {
  score: PropTypes.func,
  nickName: PropTypes.func
};


export default Score;
