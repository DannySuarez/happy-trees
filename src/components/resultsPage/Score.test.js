import React from 'react';
import { shallow } from 'enzyme';
import Score from './Score';

describe('Score Input', () => {
  it('renders Score component', () => {
    const wrapper = shallow(<Score />);
    expect(wrapper).toMatchSnapshot();
  });
});
