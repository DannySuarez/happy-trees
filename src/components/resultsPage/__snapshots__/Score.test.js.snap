// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Score Input renders Score component 1`] = `
<div>
  <h3 />
  <h3 />
</div>
`;
